experimental/python/databricks/bundles/jobs/_models/job.py

```diff
@@ -277,6 +297,7 @@ class JobDict(TypedDict, total=False):
     tasks: VariableOrList[TaskParam]
     """
     A list of task specifications to be executed by this job.
+    If more than 100 tasks are available, you can paginate through them using :method:jobs/get. Use the `next_page_token` field at the object root to determine if more results are available.
```

Comment:
This is a problem with bundle schema, we should annotate API-only descriptions and strip them.